### DISCUSSION TITLE
cluster operator keep restarting because of oomemory

### DISCUSSION THREAD

**Discussion by Sindvero** [strimzi/strimzi-kafka-operator#5041] [Q&A]
_Please use this to only for bug reports. For questions or when you need help, you can use the [GitHub Discussions](https://github.com/strimzi/strimzi-kafka-operator/discussions), our [#strimzi Slack channel](https://slack.cncf.io/) or out [user mailing list](https://lists.cncf.io/g/cncf-strimzi-users/topics)._

**Describe the bug**
The limit memory request is 384Mi which is pretty low for our use case since we have 5 brokers running and 5 ZKs which is a minimum for our use case. So my questions were why is the limit so low?, and can I increase it without problem and also create a PR with a higher memory limit? 

**To Reproduce**
Just deploy the cluster with 5 kafka brokers and 5 ZKs in an istio mesh

**Expected behavior**
Not restarting and have a bigger memory limit

**Environment (please complete the following information):**
 - Strimzi version: 0.23.0 (same issue with 0.21.1)
 - Installation method: YAML files
 - Kubernetes cluster: Kubernetes 1.19
 - Infrastructure: On aws deployed with Kops

**YAML files and logs**
Please find the configuration attached but it's basically the default one
[report-27-05-2021_02-21-44.zip](https://github.com/strimzi/strimzi-kafka-operator/files/6550688/report-27-05-2021_02-21-44.zip)

**Comment by LiamClarkeNZ** [comment]
Hi @Sindvero, 

What was restarting? The brokers or the cluster operator?

**Comment by Sindvero** [comment]
Hi @LiamClarkeNZ,

The cluster operator (strimzi cluster operator)keep restarting because of a OOMKilled. So I supposed I could increase the limit by myself right? but I was wondering why the limit is so low? and 384Mi doesn't seem like a "random" value so I was wondering how did you chose this limit?

Thanks

**Comment by LiamClarkeNZ** [comment]
Ah, gotcha, sorry, was in the title. I'm rather new to Strimzi, so unsure why it was set to 384. @scholzj may be able to speak to the reasons behind that. 

But yep, you can definitely set it higher for your deployment.

**Comment by scholzj** [comment]
384Mi is something what seems to work for most people. So I do not think we have any plans to change it. However, it is of course possible that it does not work for your particular case. You can of course always change it in your deployment just by editing the resources in the YAML / Kubernetes resource.
